src/main/java/thestonedturtle/partypanel/PartyPanelPlugin.java

```diff
@@ -62,6 +61,7 @@ public class PartyPanelPlugin extends Plugin
 	@Inject
 	private ClientToolbar clientToolbar;
 
+	@Getter
 	@Inject
 	private PartyPanelConfig config;
```

Comment:
Is there a reason making a getter for the whole config here is better than extracting the boolean out like I had done previously? Just want to understand if there's something about this approach I'm not understanding.

Reply:
I'm not sure if there's really a difference worth noting, but the main reason I made this change is because the only use for the `autoExpandButton` variable was to store a local reference for the config value. Since we can just grab the value off the config I just saw this as a redundant variable and it could potentially desync from the true config value.

I could have created a function instead of adding `@Getter` here that looked something like this to accomplish the same goal without exposing the config or using a local variable:
```java
public boolean getAutoExpandMembersValue()
{
  return config.autoExpandMembers();
}
```

The other thing is if there's ever another config option we need in a file that the plugin is passed to we can access it without any code changes.

The reason the other config has a variable is because the config could be disabled but they start the plugin while in a party so the button was added. If they then enabled the config we could end up adding a duplicate button so just basing it off the config wasn't enough.